architecture/service_catalog/index.adoc

```diff
@@ -87,6 +87,17 @@ This infrastructure allows a loose coupling between applications running in
 those services to focus on its own business logic while leaving the management
 of these services to the provider.
 
+=== Deleting resources
+
+When a user is done with a service, the service instance can be deleted. In
```

Comment:
Why would users want to delete resources? To free up memory resources? If they're not deleted, what can be expected?